package.json

```diff
@@ -12,7 +12,7 @@
     "flow:install": "flow-typed install",
     "lint": "flow && eslint --ext .js,.jsx app/javascript spec/javascripts",
     "test": "karma start --single-run",
-    "jest": "jest",
+    "jest": "jest --updateSnapshot",
```

Comment:
⚠️I'm not sure about this, I think it will make the snapshot check useless. The snapshots will be updated every time by CircleCI, therefore they will _always_ be correct. I think the idea of the snapshots is that _we_ manually take care of them. But I can be wrong,  @didierofrivia ?

Reply:
The problem is right now we are supposed to update them manually and we are forgetting, tests are failing when already merged in master.
If we add it here, they will fail in our PR's, when circle ci runs `npm run jest`, not in master.

Reply:
I think it's the other way around, if CircleCI updates the snapshot it will never fail. It runs `jest` as a part of the javascript pipeline

Reply:
We should ***never*** auto update the templates. We should first identify why it didn't failed in the PR merged and then act accordingly.